app/templates/components/forms/wizard/sessions-speakers-step.hbs

```diff
@@ -61,11 +61,11 @@
       {{#each microlocations as |microlocation|}}
         <div class="{{if device.isMobile 'grouped'}} fields">
           <div class="{{unless device.isMobile 'three wide'}} field">
-            {{input type='text' value=microlocation.name placeholder='Name'}}
+            {{input type='text' id="microlocation" value=microlocation.name placeholder='Name'}}
           </div>
           <div class="{{unless device.isMobile 'four wide'}} field">
             <div class="ui action input fluid">
-              {{input type='number' name='microlocation' value=microlocation.floor placeholder='Floor'}}
+              {{input type='number' id='microlocation_floor' value=microlocation.floor placeholder='Floor'}}
```

Comment:
Alright then no need for number validation, I didn't saw the input type is already defined as `number`.

Reply:
@kushthedude Done .